src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+
+    public void handler() {
```

Comment:
```suggestion
    public void handle() {
```
method names should be verbs